addons/web/static/src/views/list/list_renderer.js

```diff
@@ -832,7 +832,10 @@ export class ListRenderer extends Component {
 
     getFormattedValue(column, record) {
         const fieldName = column.name;
-        if (column.options.enable_formatting === false) {
+        if (
+            column.options.enable_formatting === false &&
+            !["integer", "float"].includes(column.fieldType)
+        ) {
             return record.data[fieldName];
```

Comment:
That is, I would do something like this instead:
```suggestion
        if (column.options.enable_formatting === false) {
            const value = record.data[fieldName];
            return ["integer", "float"].includes(column.fieldType) === false ? "" : value;
```

But let's see what framework js thinks about that.

Reply:
Yes but I have changes the logic to be

return ["integer", "float"].includes(column.fieldType) && !value ? "" : value;

As we need "" only when value is falsy :thinking: Thanks for the suggestion

Reply:
yep indeed, my bad, I let some words in my keyboard. 😅